subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/InputFileSnapshotNormalizationStrategy.java

```diff
@@ -23,7 +23,7 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-public enum TaskFilePropertySnapshotNormalizationStrategy implements SnapshotNormalizationStrategy {
+public enum InputFileSnapshotNormalizationStrategy implements SnapshotNormalizationStrategy {
```

Comment:
I think we should reflect that this is about path sensitivity. How about `InputPathNormalizationStrategy`? We could also rename `PathSensitivity` to `PathNormalization`.